### PR TITLE
Swift and ObjC cannot handle JSON Obj with null value

### PR DESCRIPTION
**Pull request by lott-ai**
How to reproduce

```
var snippet = new HttpSnippet({
  "url": "http://mockbin.com/har",
  "method": "POST",
  "headers": [{
    "name": "content-type",
    "value": "application/json"
  }],
  "postData": {
    "params": [],
    "text": "{\"foo\":null}",
    "stored": true,
    "mimeType": "application/json",
    "size": 0,
    "jsonObj": {
      "foo":null
    },
    "paramsObj": false
  }
});

snippet.convert("swift");
snippet.convert("objc");
```

Since "foo" is null you will receive "Uncaught TypeError: Cannot read property 'toString' of null";

Added a try-catch around https://github.com/Mashape/httpsnippet/blob/master/src/targets/swift/helpers.js#L77 and https://github.com/Mashape/httpsnippet/blob/master/src/targets/objc/helpers.js#L67.

Also added a test for swift & objc and output tests for the request.

### CODE REVIEW

**Comment by ahmadnassri** [comment]
excellent, thanks for this!

File: src/targets/objc/helpers.js [hunk 1]
```diff
@@ -64,7 +64,11 @@ module.exports = {
       case '[object Boolean]':
         return value ? '@YES' : '@NO'
       default:
-        return '@"' + value.toString().replace(/"/g, '\\"') + '"'
+        try {
+          return '@"' + value.toString().replace(/"/g, '\\"') + '"'
```
Comment (ahmadnassri) [inline src/targets/objc/helpers.js]: wouldn't this be better if it was a `null` check rather than a try/fail?

Comment (lott-ai) [reply src/targets/objc/helpers.js]: Yes of course ```if (value !== null && value !== undefined) {}``` will also work if you prefer

Comment (lott-ai) [reply src/targets/objc/helpers.js]: Its probably more efficient too, so I will change this

**Comment by lott-ai** [comment]
@ahmadnassri any updates on this?

**Pull request merged**